.travis.yml

```diff
@@ -1,5 +1,39 @@
 sudo: required
 dist: trusty
+
+env:
+  COMPOSE_VERSION: 1.24.1
+
+before_install:
+  # Docker Compose Install
+  - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  # Download and install Docker libs
+  - curl -L https://github.com/Ortus-Solutions/docker-buildfiles/archive/master.zip > docker.zip
+  - unzip docker.zip -d workbench
+  - mv workbench/docker-buildfiles-master workbench/docker
```

Comment:
Since it looks like you were able to comment up all the `box` tooling & still get it to work, I bet you can remove these three lines too (starting with the `curl`).  These seem to be setting up the box tooling, and it seems like we don't need any of that to actually run docker / docker-compose.